src/common/util/array-move.ts

```diff
@@ -20,20 +20,33 @@ function findNestedItem(
   }, obj);
 }
 
+function updateNestedItem(obj: any, path: ItemPath): any {
+  const lastKey = path.pop()!;
+  const parent = findNestedItem(obj, path);
+  parent[lastKey] = Array.isArray(parent[lastKey])
+    ? [...parent[lastKey]]
+    : [parent[lastKey]];
```

Comment:
```suggestion
    ? [...parent[lastKey]]
    : { ...parent[lastKey] };
```
Should it be this?

Reply:
They idea is to fix it here if it isn't an Array, it actually should be an array but cannot be a array because of yaml